### PR TITLE
Convert keyword maps to static initializers

### PR DESCRIPTION
**Pull request by arcady-lunarg**
Rather than having a function that needs to be called from ShInitialize to initialize some global costants, convert them to use static initializers. This also ends up saving a bit of space in the binary, especially when building with GCC.

### CODE REVIEW

File: glslang/HLSL/hlslScanContext.cpp [hunk 1]
```diff
@@ -494,49 +476,41 @@ void HlslScanContext::fillInKeywordMap()
...
-    (*SemanticMap)["SV_GROUPTHREADID"] =          EbvLocalInvocationId;
-    (*SemanticMap)["SV_GROUPINDEX"] =             EbvLocalInvocationIndex;
-    (*SemanticMap)["SV_GROUPID"] =                EbvWorkGroupId;
-    (*SemanticMap)["SV_DOMAINLOCATION"] =         EbvTessCoord;
-    (*SemanticMap)["SV_DEPTH"] =                  EbvFragDepth;
-    (*SemanticMap)["SV_COVERAGE"] =               EbvSampleMask;
-    (*SemanticMap)["SV_DEPTHGREATEREQUAL"] =      EbvFragDepthGreater;
-    (*SemanticMap)["SV_DEPTHLESSEQUAL"] =         EbvFragDepthLesser;
-    (*SemanticMap)["SV_STENCILREF"] =             EbvFragStencilRef;
+#if 0
```
Comment (jeremy-lunarg) [inline glslang/HLSL/hlslScanContext.cpp:479]: Did you intend to remove this block of code?

**Review by jeremy-lunarg — APPROVED** [review]
I left a question about a block of dead code but otherwise looks good.

File: glslang/HLSL/hlslScanContext.cpp [hunk 2]
```diff
@@ -494,49 +476,41 @@ void HlslScanContext::fillInKeywordMap()
...
     // stage, it would just be ignored as it is likely there as part of an output struct from one stage
     // to the next
-    bool bParseDX9 = false;
-    if (bParseDX9) {
-        (*SemanticMap)["PSIZE"] = EbvPointSize;
-        (*SemanticMap)["FOG"] =   EbvFogFragCoord;
-        (*SemanticMap)["DEPTH"] = EbvFragDepth;
-        (*SemanticMap)["VFACE"] = EbvFace;
-        (*SemanticMap)["VPOS"] =  EbvFragCoord;
-    }
```
Comment (arcady-lunarg) [inline glslang/HLSL/hlslScanContext.cpp:504]: This is the block of code I am preserving with `#if 0`

File: glslang/HLSL/hlslScanContext.cpp (hunk 1 above)
Comment (arcady-lunarg) [reply glslang/HLSL/hlslScanContext.cpp:479]: It was de-facto commented out in the original, so I am preserving that.

**Pull request merged**